README.md

```diff
@@ -109,11 +78,70 @@ s3://mybucket/aws-instance-scheduler/v1.4.0/instance-scheduler.template
 s3://mybucket/aws-instance-scheduler/v1.4.0/instance-scheduler-remote.template
 ```
 
-## Deploy
+### Deploy
 
 See the [AWS Instance Scheduler Implementation Guide](https://s3.amazonaws.com/solutions-reference/aws-instance-scheduler/latest/instance-scheduler.pdf) for deployment instructions, using the link to the instance-scheduler.template from your bucket, rather than the one for AWS Solutions. Ex. https://mybucket.s3.amazonaws.com/aws-instance-scheduler/v1.4.0.mybuild/instance-scheduler.template
 
-## CDK Documentation
+
+# Testing the Solution
+
+## Running Tests Locally
+to test the solution you will need to install tox
```

Comment:
Capitalization, maybe put this in a prereqs section

Reply:
maybe, though I don't think running one extra pip command is enough to justify an extra prereq section. You could also just think of it as "run command a, then command b" and it wouldn't be all that inaccurate

Reply:
The way I'm thinking about it, you're providing instructions on a task. That task is "running tests locally". In order to do that task, all I need to do is `tox`. If I want to make a change, then rerun the tests, I don't need to reinstall tox. I just run tox again. That means to me that installing tox is not a step in performing this task but a prereq. As is having python installed etc. That probably seems obvious to you and I because we know what these tools do and how they work, but it's not necessarily obvious to someone reading this for the first time.